vnext/Desktop/FeatureGate.cpp

```diff
@@ -0,0 +1,21 @@
+#include <FeatureGate.h>
+
+#include <unordered_set>
+
+using std::string;
+
+namespace {
+std::unordered_set<string> g_featureGates;
+}
+
+namespace Microsoft::React {
+
+void SetFeatureGate(string &&name) noexcept {
+  g_featureGates.insert(std::move(name));
```

Comment:
>g_featureGates [](start = 2, length = 14)

Please make sure that access to global mutable variables is always done under a mutex. Otherwise it can cause random crashes and difficult to find bugs.